OpenSDS_Scheduler_Enhancement_Design .md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Status: Pending
+
+Version: Beta
+
+Author(s): PengYiOfOpenSDS
+
+## Summary
+
+Scheduler enhancement
```

Comment:
@PengYiOfOpenSDS You are modifying the PR for extend volume for scheduler enhancement.  You should create a complete different branch to write the new PR.  Thanks.

Reply:
Also Scheduler enhancement is a big feature. We need a lot more details on the design than what is currently in the doc.